tools/postcss.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const pify = require('pify');
+const postcss = require('postcss');
+const autoprefixer = require('autoprefixer');
+const pxtorem = require('postcss-pxtorem');
+const cssVars = require('postcss-css-variables');
```

Comment:
Can we rename this `const`? It's a bit too close to the `cssvars` flag passed to the exported function!